js/pages/homePage.js

```diff
@@ -108,7 +108,7 @@ export const homePage = async () => {
         location.search = cleanSearchStr; // Page reload with clean url
       }
       
-        firebaseSignInRender({ui, account:{type:'magicLink', value:''}});
+        firebaseSignInRender({ui, account:{type:'magicLink', value:''}, usGov, signInConfig, signInConfigDev});
```

Comment:
I would move `usGov` to `shared.js` and also just import the sign in configs in `shared.js` instead of passing them here as parameters